packages/turf-isolines/bench.js

```diff
@@ -2,7 +2,7 @@ import fs from "fs";
 import path from "path";
 import load from "load-json-file";
 import Benchmark from "benchmark";
-import matrixToGrid from "matrix-to-grid";
+import matrixToGrid from "./lib/matrix-to-grid";
```

Comment:
what's the purpose of using these internal modules? That is, why not using the npm versions?

Reply:
For matrix-to-grid the file committed to turf differs from the one in npm. Looks like @DenisCarriere included a tweak when it was originally added? See https://github.com/Turfjs/turf/blame/2faafab9162bcadd35a79ec329f578881038b5ec/packages/turf-isolines/lib/matrix-to-grid.js#L45

More than happy to use the NPM version. Do you want to roll in that change and re-release? I'm not sure what the original justification was though.

For now will go with the internal modules and we can refactor this depending on what you'd like to do.